src/containers/OversiktContainer.tsx

```diff
@@ -42,6 +37,13 @@ function toReadableString(overviewTabType: OverviewTabType): string {
 const OversiktContainer = ({ tabType }: Props): ReactElement => {
   const personregisterQuery = usePersonregisterQuery();
   const personoversiktQuery = usePersonoversiktQuery();
+  const [feedbackDate, _] = useLocalStorageState<Date | null>(
```

Comment:
Tror du bare kan droppe `_` for å slippe warning her (og i `<Flexjar>`), https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#ignoring_some_returned_values